src/commands/blmove.json

```diff
@@ -7,6 +7,7 @@
         "arity": 6,
         "function": "blmoveCommand",
         "command_flags": [
+            "FAST",
```

Comment:
Blocking commands can take a long time so maybe they should not be marked as FAST?

Reply:
right, that's true, i guess i was influenced by bzpopmax (it's tagged FAST)